gctscript/wrappers/gct/gctwrapper_test.go

```diff
@@ -186,18 +187,12 @@ func TestExchangePairs(t *testing.T) {
 func TestAccountInfo(t *testing.T) {
```

Comment:
🐒 Nitpick, since you're inside it already
```suggestion
func TestExchangeAccountInfo(t *testing.T) {
```